bool StrgPos1
bool StrgPos2
bool StrgPos3
bool StrgPosGap
bool StrgPos4
bool StrgPos5
bool StrgPos6
bool StrgHorStart
bool StrgHorStop
bool StrgHorRef
---
string status
